### PR TITLE
Add thewrap.com support

### PR DESCRIPTION
**Pull request by kimkha**
solves #379 

cc @iamadamdev

### CODE REVIEW

File: src/js/contentScript.js [hunk 1]
```diff
@@ -414,6 +414,15 @@ if (matchDomain('techinasia.com')) {
...
 
+if (matchDomain('thewrap.com')) {
+    const embed = document.querySelector('.embed');
+    if (embed) {
+        // Display feature video
+        const container = document.querySelector('.featured-image-container');
+        removeDOMElement(container);
+        embed.classList.remove('d-none');
+    }
+}
```
Comment (iamadamdev) [inline src/js/contentScript.js:425]: nit: add a linebreak after }

Comment (kimkha) [reply src/js/contentScript.js:425]:
```suggestion
}

```

**Comment by iamadamdev** [comment]
Readme?

**Comment by kimkha** [comment]
Haha, mostly forgot.

**Pull request merged**